src/test/java/it/pagopa/pn/bff/mappers/tosprivacy/TosPrivacyMapperTest.java

```diff
@@ -0,0 +1,36 @@
+package it.pagopa.pn.bff.mappers.tosprivacy;
+
+import it.pagopa.pn.bff.generated.openapi.msclient.user_attributes.model.Consent;
+import it.pagopa.pn.bff.generated.openapi.msclient.user_attributes.model.ConsentAction;
+import it.pagopa.pn.bff.generated.openapi.server.v1.dto.BffConsent;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class TosPrivacyMapperTest {
+    @Test
+    void testConsentMapper() {
+        Consent consent = new Consent();
```

Comment:
Use the mock and test that is equal to the mapped object (see api keys mapper test)